<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Por favor, ingresa el número de corrida a la que quieres cambiar su estado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>511</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Actualizar estado</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_estadoCorrida">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>196</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Estado</string>
   </property>
   <item>
    <property name="text">
     <string>ACTIVA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>INACTIVA</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>80</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>114</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_actualizar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background: #1061C4;
	color:WHITE;
	border:none;
	border-radius: 6px;
	font-weight: bold;
	font-size:15px;
}

QPushButton:hover {
    background-color: #0D4FAB;    
}

QPushButton:pressed {
    background-color: #0A3F8A;     
}</string>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numeroCorrida">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="placeholderText">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_estadoActual">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Estado Actual: N/A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_cancelar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	background-color: rgb(237, 51, 59);
	color:WHITE;
	border:none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}

QPushButton::hover{
	background-color: rgb(191, 41, 49);
}

QPushButton::pressed{
	background-color: rgb(141, 30, 36);
}</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
